analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationDispatcher.cs

```diff
@@ -44,6 +44,7 @@ internal static class OperationDispatcher
         { OperationKindEx.ArrayElementReference, new ArrayElementReference() },
         { OperationKindEx.AnonymousObjectCreation, new Creation() },
         { OperationKindEx.Await, new Await() },
+        { OperationKindEx.CaughtException, new CaughtException() },
```

Comment:
Let's add repro test cases to the rule as well.

Reply:
I forgot to ask for the same thing in #6110, can you add one here?

Reply:
I dont' see a test case for the #6110

Reply:
Is this what you wanted? b47f979a870231f6cab6d3596ab3bd508fbb5b0f

Reply:
Yes, exactly. Thanks